ed_sensor_integration/src/laser/updater.cpp

```diff
@@ -321,11 +321,16 @@ void LaserUpdater::update(const ed::WorldModel& world, const std::vector<float>&
         {
             filtered_sensor_ranges[i] = sensor_ranges[i - 1];
         }
+        else
+        {
+            filtered_sensor_ranges[i] = sensor_ranges[i];
+        }
```

Comment:
How does this impact the overal behaviour?

Reply:
Increases the amount of memory used and adds a lot of assignments. I don't think its crippling for the performance but it is subject to optimization.